backend/game/cards/discard_cards/health_manipulation_cards/test_fire_blast.py

```diff
@@ -0,0 +1,13 @@
+from game.cards.discard_cards.health_manipulation_cards.fire_blast import FireBlast
+
+
+def test_fire_blast_subtracts_2_health(player, five_players):
+    for other_players in five_players:
+        other_players.current_health = 8
```

Comment:
Why is this for loop needed? Just assert result is 8 hp.

Reply:
for loop removed